packages/common/src/project/IpfsHttpClientLite/IpfsHttpClientLite.spec.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import {u8aConcat} from '@polkadot/util';
+import {create, IPFSHTTPClient} from 'ipfs-http-client';
```

Comment:
I think we can drop `ipfs-http-client` from the tests. Because this is deterministic we can easily set expected results